### DISCUSSION TITLE
Front Panel Key Press Pressure

### DISCUSSION THREAD

**Discussion by AlanMerry** [kachurovskiy/nanoels#133] [Q&A]
I am starting to familiarise my self with the controller.

I am noticing that quite a firm press is required to operate the keys.

Is there any way to reduce this pressure?

**Comment by kachurovskiy** [comment]
Alan, pressure is determined by the PCB switch pressure rating and front panel thickness, there's no easy way to reduce it.

**Reply by AlanMerry** [reply]
HI Maxim,

I looked at the overlay vs the button pressure and it would seem that most of the button pressure is coming from the overlay. I was looking at the option of getting some thinner overlays printed but will leave this on hold until I get everything running.
